code/datums/mood_events/needs_events.dm

```diff
@@ -0,0 +1,44 @@
+//nutrition
+/datum/mood_event/nutrition/fat
+	description = "<span class='warning'><B>I'm so fat..</B></span>\n" //muh fatshaming
+	mood_change = -4
+
+/datum/mood_event/nutrition/wellfed
+	description = "<span class='nicegreen'><span class='nicegreen'>My belly feels round and full.</span>\n"

```

Comment:
Bad HTML

Reply:
thank you for spotting that, I will fix it :)